tests/utils/test_rfree.py

```diff
@@ -109,7 +109,7 @@ def test_copy_rfree_errors(data_fmodel):
         rs.utils.copy_rfree(data_fmodel, data_fmodel)
 
     # Raise ValueError because "missing key" is missing,
-    # even though "R-free-flags" is present
+    # even though "R-free-flags" exists
```

Comment:
Had to make a change to get out of a loop of git madness, figured this was benign enough